WebContent/bottom_left_gro.jsp

```diff
@@ -1,6 +1,13 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html>
 <html>
-   <body>  
-	<table> 
+<head>
+<meta charset="ISO-8859-1">
+<title>Insert title here</title>
+</head>
+<body>
+<table> 
           <thead><h2>Groceries details</h2></thead>
```

Comment:
why you add bottom_left_gro. is this your module? u need to add only your veg module